client/app/components/composites/Menu/Menu.js

```diff
@@ -0,0 +1,102 @@
+import { Component, PropTypes } from 'react';
+import ReactDOM from 'react-dom';
+import r, { div } from 'r-dom';
+import _ from 'lodash';
+
+import MenuLabel from './MenuLabel';
+import MenuLabelDropdown from './MenuLabelDropdown';
+import MenuContent from './MenuContent';
+import css from './Menu.css';
+
+const INITIAL_MENUCONTENT_POSITION = 75;
+const INITIAL_ARROW_POSITION = 75;
+const MENUCONTENT_OVERLAP = 5;
+const MENULABEL_MAP = {
+  menu: MenuLabel,
+  dropdown: MenuLabelDropdown,
+};
+
+class Menu extends Component {
+
+  constructor(props, context) {
+    super(props, context);
+
+    _.bindAll(this, [
+      'handleMouseOver',
+      'calculateDropdownPosition',
+    ]);
```

Comment:
Fine if you can live with it, but I really don't like `_.bindAll` with method names as strings. It can lead to very subtle bugs if the string has a typo, the method name changes, or the method is removed.